docs/GettingStartedDocs/install_oe_sdk-Ubuntu_16.04.md

```diff
@@ -11,6 +11,7 @@
         - If your device does not support SGX, follow the [instructions for simulation mode](/docs/GettingStartedDocs/install_oe_sdk-Simulation.md).
```

Comment:
Perhaps it would help to mention that Open Enclave provides an easy way to discover the level of SGX support by running `/opt/openenclave/bin/oesgx`?

Reply:
done.